### PR TITLE
[presentation] crysatl0523 resigned up for time slot

### PR DESCRIPTION
**Pull request by crystal0523**
---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

File: proposal/README.rst [hunk 1]
```diff
@@ -174,8 +174,8 @@ Time Table
     - Job shop scheduling
   * - B5
     - 08:20 - 08:38
-    - TBD
-    - TBD
+    - LittleLaGi
```
Comment (yungyuc) [inline proposal/README.rst:177]: You may not touch other lines.  You seemed to mistakenly included commits cf4934a769c934f2c410de8546c5c65e1ea37bcf and 001f6d3a3f07ca372c2aae7d2cf77a9138edc9f7.  Please do not include unrelated commits.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: proposal/README.rst (hunk 1 above)
Comment (crystal0523) [reply proposal/README.rst]: Ok, I didn't notice it before, and I just modified it!

**Review by yungyuc — APPROVED** [review]
